sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/resources/Sgv2RowsResourceApi.java

```diff
@@ -0,0 +1,340 @@
+package io.stargate.sgv2.restsvc.resources;
+
+import com.codahale.metrics.annotation.Timed;
+import io.stargate.sgv2.restsvc.models.RestServiceError;
+import io.stargate.sgv2.restsvc.models.Sgv2RESTResponse;
+import io.stargate.sgv2.restsvc.models.Sgv2RowsResponse;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import java.util.List;
+import java.util.Map;
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.HeaderParam;
+import javax.ws.rs.PATCH;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.PathSegment;
+import javax.ws.rs.core.Response;
+
+/**
+ * Definition of REST API endpoint methods including both JAX-RS and Swagger annotations. No
+ * implementations.
+ *
+ * <p>NOTE: class annotations cannot be included in the interface (at least for JAX-RS) and must be
+ * included in the implementation class.
```

Comment:
`@Api` can be inherited, so maybe we can move it to the interface in order to have all the Swagger annotations in one place.

Reply:
Class annotation `@Api`? Ok, I can move that, makes sense. I only checked that JAX-RS ones cannot be moved.